backend/TRFSAE.MemberPortal.API/Controllers/ProjectController.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Mvc;
+using TRFSAE.MemberPortal.API.Interfaces;
+using TRFSAE.MemberPortal.API.Services;
+
+
+namespace TRFSAE.MemberPortal.API.Controllers;
+
+[ApiController]
+[Route("api/project")]
+public class ProjectController : ControllerBase
+{
+    private readonly IProjectService _ProjectService;
+    public ProjectController(IProjectService projectService)
+    {
+        _ProjectService = projectService;
+    }
+
+    [HttpGet]
+    public async Task<IActionResult> GetAllProjects()
+    {
+        var projects = await _ProjectService.GetAllProjectsAsync();
+        return Ok(projects);
+    }
+
+    [HttpGet("{id}")]
+    public async Task<IActionResult> GetAllProjectTasks(Guid id)
+    {
+        var tasks = await _ProjectService.GetAllProjectTasksAsync(id);
+        return Ok(tasks);
+    }
+
+    [HttpPut("{userId}/projectId")]
```

Comment:
All paths should be lowercase for endpoints. You can either do `project-id` or `project_id`, but please do not use `projectId`. 

just as a side note i don't think this is a very useful endpoint lol, who told you to make it